vcsim/main.go

```diff
@@ -107,7 +113,12 @@ func main() {
 	switch flag.Arg(0) {
 	case "uuidgen": // util-linux not installed on Travis CI
 		fmt.Println(uuid.New().String())
-		return
+		os.Exit(0)
+	case "version":
+		fmt.Fprintf(flag.CommandLine.Output(), "Build Version: %s\n", buildVersion)
```

Comment:
I hadn't seen `flag.CommandLine.Output()` before but looks like it defaults to `stderr`.. we should use `stdout` unless printing an error/warning/etc.

Reply:
Good catch, copy leftover from the previous (help) proposal.